youtube_dl/YoutubeDL.py

```diff
@@ -2261,11 +2263,11 @@ def list_formats(self, info_dict):
     def list_thumbnails(self, info_dict):
         thumbnails = info_dict.get('thumbnails')
         if not thumbnails:
-            self.to_screen('[info] No thumbnails present for %s' % info_dict['id'])
+            self.to_screen('%s has no thumbnail' % info_dict['id'])
```

Comment:
Should this have lost its `[info]`?

Should the same message be used (the severity being supplied by `report_warning()` at l.2448?

Reply:
I thought it was strange to put `[info]` here since it doesn't list any meaningful info.
And warning is too strong when users just want to get list.
As to `--write-...`, it should be warned because users don't get the results they directed or expected, I think.